src/main/kotlin/com/cognifide/gradle/aem/instance/LocalHandle.kt

```diff
@@ -35,7 +35,10 @@ class LocalHandle(val project: Project, val sync: InstanceSync) {
     val dir = File("${config.instancesPath}/${instance.name}")
 
     val jar: File by lazy {
-        FileOperations.find(dir, listOf("cq-quickstart*.jar")) ?: File(dir, "cq-quickstart.jar")
+        FileOperations.find(dir, "*cq-quickstart*.jar")
+                ?: FileOperations.find(dir, "*quickstart*.jar")
+                ?: FileOperations.find(dir, "*.jar")
```

Comment:
and this as a fallback (a little bit error prone if more than 2 jars will be in instance files dir / probably never... :)